File: sub-model/src/main/java/org/github/base/service/impl/CategoryServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.CategoryEntity;
import org.github.base.mapper.ICategoryMapper;
import org.github.base.service.ICategoryService;
import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 商品分类  服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-03-27
 */
@Service
public class CategoryServiceImpl extends ServiceImpl<ICategoryMapper, CategoryEntity> implements ICategoryService {}

File: sub-mysql-blog-service/src/main/java/org/github/mysql/blog/base/service/impl/CategoryServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.blog.base.service.impl;

import org.github.mysql.blog.base.entity.CategoryEntity;
import org.github.mysql.blog.base.mapper.ICategoryMapper;
import org.github.mysql.blog.base.service.ICategoryService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 *  服务实现类
 *
 * @author JYD_XL
 */
@Service
open class CategoryServiceImpl : ServiceImpl<ICategoryMapper, CategoryEntity>(), ICategoryService
